examples/navegg-add.amp.html

```diff
@@ -0,0 +1,53 @@
+<!doctype html>
```

Comment:
you don't need to add an example page, please only add the example to `examples/ads.amp.html`